types/authorization.yaml

```diff
@@ -32,6 +32,7 @@ types:
       - Processing
       - Accepted
       - Rejected
+      - TokenTimedOut
```

Comment:
`TimedOut` might be enough here as its type is already called `TokenValidationStatus` and hence one can already deduce that the timeout is related to the token.

Reply:
Renamed :)